MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -70,6 +72,36 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
             setScreenOrientation(requestParam: requestParam, callbackId: callbackId)
         case .getAccessToken:
             fetchTokenDetails(callbackId: callbackId)
+        case .loadAd:
+            loadRequestedAd(forParams: requestParam) { [weak self] result in
+                switch result {
+                case .success:
+                    self?.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: "Ad loaded")
+                case .failure(let error):
+                    self?.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: error.localizedDescription)
+                }
+            }
```

Comment:
Can we move this to separate method ?

Reply:
It's already a method, but I see what you mean, I'll make this clearer